packages/wallets/src/components/WalletListCardActions/WalletListCardActions.tsx

```diff
@@ -93,7 +88,7 @@ const WalletListCardActions: React.FC<TProps> = ({ isActive, isDemo, loginid })
                     icon={button.icon}
                     key={button.name}
                     onClick={() => {
-                        switchAccount(loginid);
+                        loginid && switchAccount(loginid);
```

Comment:
Question: Should we wait till we switch the account then go to /cashier/... ?

Reply:
Good point. I believe that was previously added so that we could redirect to the cashier pages even from an inactive wallet account in the wallet accordion. But since we're only displaying one wallet at a time currently we wouldn't need it. Removed unneeded logic 👍